### PR TITLE
hi

### PR DESCRIPTION
**Pull request by PaulGithub15**
im a tropical cyclone fan and i would like to propose a change, regarding [cyclone simulator](https://monsoonjr99.github.io/cyclone-sim/).

### CODE REVIEW

**Comment by capJavert** [comment]
@PaulGithub15 

I would love to merge this but you will need to do the following:
- JSON format is not valid, you can use https://jsonlint.com/ to validate your JSON syntax and look at other submissions for examples

Thanks!

File: clippy.json [hunk 1]
```diff
@@ -27,6 +27,15 @@
         "I am programmed to reflexively shield my eyes from Assembly.",
         "Source control is a magical thing.",
         "Use your brain, use Rust."
+            ],
+    ],
+    "https://monsoonjr99.github.io/cyclone-sim/": [ 
+        "WATCH OUT PHILIPPINES!",
+        "wow typhoon good very storm",
+        "wow hurricane good very storm",
+        "if damage = 1b dollar or more, retire name remember that",
```
Comment (capJavert) [inline clippy.json]: not sure if this one is gramatically correct, can you update as well?

Comment (PaulGithub15) [reply clippy.json]: those 2 very storm stuffs, i did that on purpose (I did doge thing)

Comment (PaulGithub15) [reply clippy.json]: also the 1b dollar or more thing, it was a note to self

Comment (PaulGithub15) [reply clippy.json]: also the watch out philippines thing, some strong typhoons might make landfall in my home country (I think u know that) too.

Comment (capJavert) [reply clippy.json]: Yeah its all god I only do not understand what "retire name remember that" means, it looks like typo or you choose a wrong word.

Comment (PaulGithub15) [reply clippy.json]: retire name: replace if costly/deadly

Comment (capJavert) [reply clippy.json]: Not sure still its widely known, I removed it and merged other ones. Thank you for your contributions 👍

**Review by capJavert — APPROVED** [review]

**Pull request merged**